src/core/src/op/topk.cpp

```diff
@@ -486,3 +480,42 @@ bool op::v3::TopK::has_evaluate() const {
 
     return true;
 }
+
+// =============== V11 ===============
+ov::op::v11::TopK::TopK(const Output<Node>& data,
+                        const Output<Node>& k,
+                        const int64_t axis,
+                        const std::string& mode,
+                        const std::string& sort,
+                        const element::Type& index_element_type,
+                        const bool stable)
+    : TopK(data, k, axis, as_enum<Mode>(mode), as_enum<SortType>(sort), index_element_type, stable) {}
+
+ov::op::v11::TopK::TopK(const Output<Node>& data,
+                        const Output<Node>& k,
+                        const int64_t axis,
+                        const Mode mode,
+                        const SortType sort,
+                        const element::Type& index_element_type,
+                        const bool stable)
+    : op::v3::TopK{data, k, axis, mode, sort, index_element_type},
+      m_stable{stable} {}
```

Comment:
I think it makes sense to check if specified attributes are compatible like `stable` and `sort`

Reply:
I thought about it too but decided to go the other way. Instead of being very strict we could let the graph be constructed with `stable == true` and `sort` equal to `indices` or `none` and simply ignore the `stable` attribute in such cases. 
This way the graph can still be constructed (import/conversion will not fail) and the implementation will work according to the spec.

Will that work or should I add a check?

Reply:
Check added but it required a lot refactor of the TopK operators.